HeavyFlavorAnalysis/RecoDecay/BuildFile.xml

```diff
@@ -0,0 +1,27 @@
+<use   name="DataFormats/ParticleFlowCandidate"/>
+<use   name="DataFormats/PatCandidates"/>
+<use   name="DataFormats/RecoCandidate"/>
+<use   name="DataFormats/TrackReco"/>
+<use   name="DataFormats/VertexReco"/>
+<use   name="FWCore/Framework"/>
+<use   name="FWCore/MessageLogger"/>
+<use   name="PhysicsTools/CandUtils"/>
+<use   name="RecoVertex/KalmanVertexFit"/>
+<use   name="RecoVertex/KinematicFit"/>
+<use   name="RecoVertex/KinematicFitPrimitives"/>
+<use   name="RecoVertex/VertexPrimitives"/>
+<use   name="TrackingTools/PatternTools"/>
+<use   name="TrackingTools/Records"/>
+<use   name="TrackingTools/TransientTrack"/>
+<flags CXXFLAGS="-Wno-delete-non-virtual-dtor"/>
```

Comment:
this was added to remove warnings coming from other packages and making
harder to spot warnings from my code...

On Tue, 23 Aug 2016, Slava Krutelyov wrote:

> 
> In HeavyFlavorAnalysis/RecoDecay/BuildFile.xml:
> 
> > +<use   name="DataFormats/ParticleFlowCandidate"/>
> > +<use   name="DataFormats/PatCandidates"/>
> > +<use   name="DataFormats/RecoCandidate"/>
> > +<use   name="DataFormats/TrackReco"/>
> > +<use   name="DataFormats/VertexReco"/>
> > +<use   name="FWCore/Framework"/>
> > +<use   name="FWCore/MessageLogger"/>
> > +<use   name="PhysicsTools/CandUtils"/>
> > +<use   name="RecoVertex/KalmanVertexFit"/>
> > +<use   name="RecoVertex/KinematicFit"/>
> > +<use   name="RecoVertex/KinematicFitPrimitives"/>
> > +<use   name="RecoVertex/VertexPrimitives"/>
> > +<use   name="TrackingTools/PatternTools"/>
> > +<use   name="TrackingTools/Records"/>
> > +<use   name="TrackingTools/TransientTrack"/>
> > +<flags CXXFLAGS="-Wno-delete-non-virtual-dtor"/>
> 
> CXXFLAGS use should be justified.
> suppressing a warning doesn't look like a good solution
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the
> thread.[AE4A-WnPZeb69EX6ufdkW-ariJcb_dAEks5qixYhgaJpZM4JlnI8.gif]
> 
> 
>